composer.json

```diff
@@ -1,6 +1,7 @@
 {
-  "name": "geekwright/regdom",
+  "name": "xoops/regdom",
   "description": "OO adaptation of Florian Sager's regdom library for querying Mozilla Public Suffix List",
+  "version": "1.0.10",
```

Comment:
I would suggest 2.0.0-alpha-1 to start with. I usually just specify it in the repository tags instead.
[https://getcomposer.org/doc/04-schema.md#version](url)